src/network/network.rs

```diff
@@ -55,6 +55,9 @@ pub struct NetworkOptions {
     #[serde(rename = "networks")]
     pub networks: HashMap<String, PerNetworkOptions>,
 
+    #[serde(rename = "network_info")]
+    pub network_info: HashMap<String, Network>,
```

Comment:
We either need to make this optional or if it is mandatory this needs to be added into test stub data.